doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -289,7 +289,34 @@ Networking samples
 NFC samples
 -----------
 
-|no_changes_yet_note|
+* :ref:`record_launch_app` sample:
+
+  * Added support for the :ref:`zephyr:nrf54l15pdk_nrf54l15` board.
+  * Added support for the :ref:`zephyr:nrf54h20dk_nrf54h20` board.
+
+* :ref:`record_text` sample:
+
+  * Added support for the :ref:`zephyr:nrf54l15pdk_nrf54l15` board.
+  * Added support for the :ref:`zephyr:nrf54h20dk_nrf54h20` board.
```

Comment:
I am not sure if the nRF54h support should be added now in the release notes for the devtag release.
@FrancescoSer : Could you please confirm this.

Reply:
As discussed offline, let's leave this in.